tests/integration/targets/rds_option_group/tasks/main.yml

```
@@ -0,0 +1,243 @@
+---
+- name: rds_option_group tests
+  collections:
+    - amazon.aws
+  
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
+
+  block:
+  - name: create a VPC
+    ec2_vpc_net:
+        name: "{{ resource_prefix }}-vpc"
+        state: present
+        cidr_block: "10.122.122.128/26"
+        tags:
+            Name: "{{ resource_prefix }}-vpc"
+            Description: "created by rds_instance integration tests"
+    register: vpc_result
+
+  - name: create subnets
+    ec2_vpc_subnet:
+        cidr: "{{ item.cidr }}"
+        az: "{{ item.zone }}"
+        vpc_id: "{{ vpc_result.vpc.id }}"
+        tags:
+            Name: "{{ resource_prefix }}-subnet"
+            Description: "created by rds_instance integration tests"
+        state: present
+    register: subnets_result
+    loop:
+        - {"cidr": "10.122.122.128/28", "zone": "{{ aws_region }}a"}
+        - {"cidr": "10.122.122.144/28", "zone": "{{ aws_region }}b"}
+        - {"cidr": "10.122.122.160/28", "zone": "{{ aws_region }}c"}
+
+  - name: Create security groups
+    ec2_group:
+        name: "{{ item }}"
+        description: "created by rds_instance integration tests"
+        state: present
+    register: sgs_result
+    loop:
+        - "{{ resource_prefix }}-sg-1"
+        - "{{ resource_prefix }}-sg-2"
+        - "{{ resource_prefix }}-sg-3"
+
+  - debug: var=sgs_result
+
+  - name: List all the option groups
+    rds_option_group_info:
+      region: "{{ aws_region }}"
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+    register: option_groups
+  
+  - debug: var=option_groups
+
+  - name: Create an RDS Mysql option group
+    rds_option_group:
+        #profile: production
+        region: "{{ aws_region }}"
+        aws_access_key: '{{ aws_access_key }}'
+        aws_secret_key: '{{ aws_secret_key }}'
+        security_token: '{{ security_token | default(omit) }}'
```

@alinabuzachis It's worth trying to get rds_option_group and rds_option_group_info added to the 2.9 groups ASAP.  2.9 drops into security-fixes only soon.

```suggestion
        region: "{{ aws_region }}"
        aws_access_key: '{{ aws_access_key }}'
        aws_secret_key: '{{ aws_secret_key }}'
        security_token: '{{ security_token | default(omit) }}'
```

---

@tremble ok, thanks. I will try doing that.